cns/errorcodeframework/error_docs.md

```diff
@@ -0,0 +1,76 @@
+# Azure CNS HTTP Error Code Framework
```

Comment:
@rbtr : I do not remember what we finalized about having design docs in code in the acn repo. Is this file ok to be here or should we move it to Eng Hub or a wiki?

Reply:
it is appropriate for this documentation to be here. it only talks about things in this opensource repo